apps/emqx_coap/etc/emqx_coap.conf

```diff
@@ -9,7 +9,7 @@
 ## Examples:
 ##    coap.bind.udp.x = 0.0.0.0:5683 | :::5683 | 127.0.0.1:5683 | ::1:5683
 ##
-coap.bind.udp.1 = 0.0.0.0:5683
+coap.bind.udp.1 = "0.0.0.0:5683"
 ##coap.bind.udp.2 = 0.0.0.0:6683
```

Comment:
quote the examples too. this applies to all other places.

Reply:
fixed